Gemfile

```diff
@@ -8,6 +8,7 @@ gemspec
 # jquery-rails is used by the dummy application
 gem "jquery-rails"
 gem "mysql2"
+gem "bootstrap-datepicker-rails", "~> 1.4.0"
```

Comment:
Since this is a gem project, its dependencies should be declared in the gemspec.